sktime/forecasting/nnetsaucemts.py

```diff
@@ -0,0 +1,666 @@
+
+    type_pi: str.
+
+        type of prediction interval; currently:
+
+        - "gaussian": simple, fast, but: assumes stationarity of Gaussian in-sample
+        residuals and independence in the multivariate case
+        - "kde": based on Kernel Density Estimation of in-sample residuals
+        - "bootstrap": based on independent bootstrap of in-sample residuals
+        - "block-bootstrap": based on basic block bootstrap of in-sample residuals
+        - "scp-kde": Sequential split conformal prediction with Kernel Density
+        Estimation of calibrated residuals
+        - "scp-bootstrap": Sequential split conformal prediction with independent
+        bootstrap of calibrated residuals
+        - "scp-block-bootstrap": Sequential split conformal prediction with basic
+        block bootstrap of calibrated residuals
+        - "scp2-kde": Sequential split conformal prediction with Kernel Density
+        Estimation of standardized calibrated residuals
+        - "scp2-bootstrap": Sequential split conformal prediction with independent
+        bootstrap of standardized calibrated residuals
+        - "scp2-block-bootstrap": Sequential split conformal prediction with basic
+        block bootstrap of standardized calibrated residuals
+        - based on copulas of in-sample residuals: 'vine-tll', 'vine-bb1',
+        'vine-bb6', 'vine-bb7', 'vine-bb8', 'vine-clayton',
+        'vine-frank', 'vine-gaussian', 'vine-gumbel', 'vine-indep', 'vine-joe',
+        'vine-student'
+        - 'scp-vine-tll', 'scp-vine-bb1', 'scp-vine-bb6', 'scp-vine-bb7',
+        'scp-vine-bb8', 'scp-vine-clayton', 'scp-vine-frank', 'scp-vine-gaussian',
+        'scp-vine-gumbel', 'scp-vine-indep', 'scp-vine-joe', 'scp-vine-student'
+        - 'scp2-vine-tll', 'scp2-vine-bb1', 'scp2-vine-bb6', 'scp2-vine-bb7',
+        'scp2-vine-bb8', 'scp2-vine-clayton', 'scp2-vine-frank', 'scp2-vine-gaussian',
+          'scp2-vine-gumbel', 'scp2-vine-indep', 'scp2-vine-joe', 'scp2-vine-student'
+
+    block_size: int.
+        size of block for 'type_pi' in ("block-bootstrap", "scp-block-bootstrap",
+        "scp2-block-bootstrap").
+        Default is round(3.15*(n_residuals^1/3))
+
+    replications: int.
+        number of replications (if needed, for predictive simulation). Default
+          is 'None'.
+
+    kernel: str.
+        the kernel to use for residuals density estimation (used for predictive
+        simulation). Currently, either 'gaussian' or 'tophat'.
+
+    agg: str.
+        either "mean" or "median" for simulation of bootstrap aggregating
+
+    seed: int.
+        reproducibility seed for nodes_sim=='uniform' or predictive simulation.
+
+    backend: str.
+        "cpu" or "gpu" or "tpu".
+
+    verbose: int.
+        0: not printing; 1: printing
+
+    show_progress: bool.
+        True: progress bar when fitting each series; False: no progress bar when
+        fitting each series
+
+    """
+
+    from nnetsauce import DeepMTS as DeepMTS0  # dangerous
```

Comment:
Please remove this line too. You are making the import later in `__init__`, so it's not required as far as I can see right now.

Reply:
Fixed